RecoEgamma/EgammaElectronProducers/interface/LowPtGsfElectronFeatures.h

```diff
@@ -0,0 +1,37 @@
+#ifndef RecoEgamma_EgammaElectronProducers_LowPtGsfElectronFeatures_h
+#define RecoEgamma_EgammaElectronProducers_LowPtGsfElectronFeatures_h
+
+#include "DataFormats/Common/interface/Ptr.h"
+#include "DataFormats/Common/interface/Ref.h"
+#include "DataFormats/Common/interface/RefToPtr.h"
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/EgammaCandidates/interface/GsfElectron.h"
+#include "RecoEcal/EgammaCoreTools/interface/EcalClusterLazyTools.h"
+#include <vector>
+
+namespace reco {
```

Comment:
I think this is breaking http://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1 rule 7:

>Do not forward-declare an entity from another package.